code/game/objects/items/storage/bags.dm

```diff
@@ -154,6 +154,12 @@
 		UnregisterSignal(listeningTo, COMSIG_MOVABLE_MOVED)
 		listeningTo = null
 
+/obj/item/storage/bag/ore/attackby(obj/item/attacking_item, mob/user, params)
+	if(istype(attacking_item, /obj/item/boulder))
+		to_chat(user, span_warning("You can't fit \the [attacking_item] into \the [src]. Perhaps you should break it down first, or find an ore box."))
```

Comment:
I think in this case, there's going to be a reasonable argument for more specificity here, since most players are probably going to treat boulders like ore unless instructed otherwise (Though since we want players to leave boulders alone to teleport them for processing, I agree it needs some kind of tweak from where it's at now).